docs/user_workloads/instancetypes.md

```diff
@@ -143,10 +143,59 @@ $ kubectl get vmis/example-preference-user-override -o json | jq .spec.domain.de
     "name": "cloudinitdisk"
   }
 ]
+```
+
+### PreferredCPUTopology
 
+A preference can optionally include a `PreferredCPUTopology` that defines how the guest visible CPU topology of the `VirtualMachineInstance` is constructed from vCPUs supplied by an instance type.
 
+```yaml
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: example-preference-cpu-topology
+spec:
+  cpu:
+    preferredCPUTopology: cores
 ```
 
+The allowed values for `PreferredCPUTopology` include:
+
+* `sockets` (default) - Provides vCPUs as sockets to the guest
+* `cores` - Provides vCPUs as cores to the guest
+* `threads` - Provides vCPUs as threads to the guest
+* `spread` - Spreads vCPUs across sockets and cores by default. See the following [SpreadOptions](#### SpreadOptions) section for more details.
+
```

Comment:
Missing `any` option ?

Reply:
@RoniKishner  Good point, I could add a comment but `Any` isn't useful in the context of `preferredCPUTopology` as it just gets translated into `sockets` if provided with an instance type.

It's actually more useful in the context of using a preference without an instance type as it allows us to define a preference with a vCPU requirement that can be expressed as any combination of sockets, cores and threads within the VM.

This is a pretty bespoke use case that I didn't want to cover just now until the docs are broken up and we have more space to cover it.

Reply:
A comment would do, some .yaml files might have it already and it would be confusing for them to see `any` and not know what it does.